core/src/main/java/com/linecorp/armeria/common/Scheme.java

```diff
@@ -78,8 +78,13 @@
         if (scheme == null) {
             return Optional.empty();
         }
-
-        return Optional.ofNullable(SCHEMES.get(Ascii.toLowerCase(scheme)));
+        final Optional<Scheme> parsedScheme = Optional.ofNullable(SCHEMES.get(Ascii.toLowerCase(scheme)));
+        if (parsedScheme.isPresent()) {
+            return parsedScheme;
+        }
+        return Optional.ofNullable(SCHEMES.get(Ascii.toLowerCase(SerializationFormat.NONE.uriText() +
+                                                                 '+' +
+                                                                 scheme)));
```

Comment:
`SerializationFormat.NONE.uriText()` is defined in lowercase. How about reusing lowercased scheme? 😀
```java
final String lowerCaseScheme = Ascii.toLowerCase(scheme);
final Optional<Scheme> parsedScheme = Optional.ofNullable(SCHEMES.get(lowerCaseScheme));
if (parsedScheme.isPresent()) {
    return parsedScheme;
}
return Optional.ofNullable(SCHEMES.get(SerializationFormat.NONE.uriText() +
                                                                 '+' +
                                                                 lowerCaseScheme)));
```

Reply:
Done.